style.css

```diff
@@ -510,8 +1014,556 @@ body {
+    font-size: 15px;
+  }
+}
+
+@media (min-width: 410px) {
+  .bottom-section__text-block {
+    display: -webkit-box;
+    display: -ms-flexbox;
+    display: flex;
+    -webkit-box-orient: vertical;
+    -webkit-box-direction: normal;
+    -ms-flex-direction: column;
+    flex-direction: column;
+    -webkit-box-align: center;
+    -ms-flex-align: center;
+    align-items: center;
+  }
+}
+
+@media (min-width: 520px) {
+  .bottom-section__text-block {
+    margin-left: 20%;
+    margin-right: 10%;
+  }
+
+  .bottom-section__image {
+    padding-left: 35px;
+    padding-right: 30px;
+  }
+}
+
+@media (min-width: 768px) {
+  .bottom-section__block {
+    margin-left: 30%;
+    margin-right: 15%;
+  }
+
+  .bottom-section__image {
+    padding-left: 45px;
+    padding-right: 40px;
+  }
+}
+
+@media (min-width: 992px) {
+  .bottom-section__block {
+    position: absolute;
+    top: 30%;
+    right: 3%;
+    padding: 30px 40px;
+    margin-left: 50%;
+    margin-right: 0;
+  }
+
+  .bottom-section__image {
+    padding: 0;
+    margin-right: 40%;
+  }
+
+  .bottom-section__img {
+    margin: 137px 0;
+  }
+
+  .bottom-section__text-block {
+    -webkit-box-align: start;
+    -ms-flex-align: start;
+    align-items: flex-start;
+  }
+}
+
+@media (min-width: 1200px) {
+  .bottom-section__block {
+    margin-right: 7%;
+  }
+
+  .bottom-section__image {
+    margin-left: 3%;
+  }
+}
+
+@media (min-width: 992px) {
+  .bottom-section__title,
+  .bottom-section__subtitle,
+  .bottom-section__description {
+    margin-left: -80px;
+  }
+
+  .bottom-section__link {
+    position: absolute;
+    right: 8%;
+    bottom: 11%;
+    padding: 0;
+    width: 345px;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 13px;
+    line-height: normal;
+  }
+
+  .bottom-section__btn {
+    display: flex;
+    margin-top: -12.5%;
+    margin-left: 30%;
+    height: 43px;
+    text-align: center;
+    justify-content: center;
+  }
+
+  .bottom-section__arrow div {
+    bottom: 7px;
+    left: 240px;
+  }
+}
+
+@media (min-width: 1200px) {
+  .bottom-section__link {
+    right: 10%;
+    bottom: 21%;
+  }
+
+  .bottom-section__btn {
+    margin-top: -12.5%;
+    margin-left: 30%;
+  }
+
+  .bottom-section__arrow div {
+    bottom: 7px;
+    left: 240px;
+  }
+}
+
+@media (min-width: 770px) and (max-width: 992px) {
+  .bottom-section__title {
+    margin-left: -65px;
+  }
+
+  .bottom-section__subtitle {
+    margin-left: -40px;
+  }
+
+  .bottom-section__btn {
+    margin-left: 75px;
+  }
+}
+
+@media (min-width: 510px) and (max-width: 770px){
+  .bottom-section__title {
+    margin-left: -270px;
+  }
+
+  .bottom-section__subtitle {
+    margin-left: -250px;
+  }
+
+  .bottom-section__description {
+    margin-left: -100px;
+  }
+}
+
+@media (min-width: 768px) and (max-width: 992px) {
+  .bottom-section__link {
+    display: inline-block;
+    margin-left: 600px;
+  }
+
+  .bottom-section__arrow {
+    display: block;
+    margin-top: 20px; 
```

Comment:
Codacy found an issue: [Expected newline after ";" in a multi-line declaration block (declaration-block-semicolon-newline-after)](https://app.codacy.com/gh/WebHeroSchool/locoalin/pullRequest?prid=5835770)